cmd/transform.go

```diff
@@ -110,6 +107,9 @@ func transformHandler(cmd *cobra.Command, flags transformFlags) {
 		}
 		startQA(flags.qaflags)
 		logrus.Debugf("Creating a new plan.")
+		if _, err := os.Stat(flags.customizationsPath); os.IsNotExist(err) {
+			flags.customizationsPath = ""
+		}
```

Comment:
This should happen only if the directory is default. Else it should be an error.

Reply:
done!